zaza/openstack/charm_tests/ceilometer/tests.py

```diff
@@ -40,41 +40,46 @@ def setUpClass(cls):
 
     @property
     def services(self):
-        """Return a list services for Openstack Release."""
+        """Return a list services for Openstack Release.
+
+        Note: disabling ceilometer-polling and ceilometer-agent-central due to
+        bug #1846390.
+        https://bugs.launchpad.net/charms/+source/ceilometer/+bug/1846390
+        """
         current_release = openstack_utils.get_os_release()
         services = []
 
         if current_release >= CeilometerTest.XENIAL_PIKE:
-            services.append('ceilometer-polling: AgentManager worker(0)')
+            # services.append('ceilometer-polling: AgentManager worker(0)')
             services.append('ceilometer-agent-notification: '
                             'NotificationService worker(0)')
         elif current_release >= CeilometerTest.XENIAL_OCATA:
             services.append('ceilometer-collector: CollectorService worker(0)')
-            services.append('ceilometer-polling: AgentManager worker(0)')
+            # services.append('ceilometer-polling: AgentManager worker(0)')
             services.append('ceilometer-agent-notification: '
                             'NotificationService worker(0)')
             services.append('apache2')
         elif current_release >= CeilometerTest.XENIAL_NEWTON:
             services.append('ceilometer-collector - CollectorService(0)')
-            services.append('ceilometer-polling - AgentManager(0)')
+            # services.append('ceilometer-polling - AgentManager(0)')
             services.append('ceilometer-agent-notification - '
                             'NotificationService(0)')
             services.append('ceilometer-api')
         else:
-            services.append('ceilometer-collector')
+            # services.append('ceilometer-collector')
             services.append('ceilometer-api')
             services.append('ceilometer-agent-notification')
 
             if current_release < CeilometerTest.TRUSTY_MITAKA:
                 services.append('ceilometer-alarm-notifier')
                 services.append('ceilometer-alarm-evaluator')
 
-            if current_release >= CeilometerTest.TRUSTY_LIBERTY:
+            # if current_release >= CeilometerTest.TRUSTY_LIBERTY:
                 # Liberty and later
-                services.append('ceilometer-polling')
-            else:
+                # services.append('ceilometer-polling')
+            # else:
                 # Juno and earlier
-                services.append('ceilometer-agent-central')
+                # services.append('ceilometer-agent-central')
```

Comment:
Hi Eduardo, In what release combination are you seeing ceilometer-agent-central end up in this list of services? I ask because trusty-mitaka should be the earliest release we're running tests for and based on the logic above ceilometer-agent-central shouldn't end up in the list for trusty-mitaka or above.

Reply:
I'm seeing that from trusty-mitaka onwards. I could change that part

Reply:
It seems like something is broken in this test code then. I'd suggest some debugging to see why ceilometer-agent-central is in the list of services for trusty-mitaka onward.